### DISCUSSION TITLE
Techonology/%E5%9F%BA%E4%BA%8EHexo%E6%A1%86%E6%9E%B6%E6%90%AD%E5%BB%BA%E4%B8%AA%E4%BA%BA%E5%8D%9A%E5%AE%A2%E8%AE%B0%E5%BD%95/

### DISCUSSION THREAD

**Discussion by giscus** [APLaS-Plus/APLaS-Plus.github.io#2] [Announcements]
# Techonology/%E5%9F%BA%E4%BA%8EHexo%E6%A1%86%E6%9E%B6%E6%90%AD%E5%BB%BA%E4%B8%AA%E4%BA%BA%E5%8D%9A%E5%AE%A2%E8%AE%B0%E5%BD%95/

操作在GitHub上部署自己的博客 Deploy your blog on GitHub 下载并安装git和node.js。  在存放博客文件的目录下新建Hexo文件夹用于下载存放Hexo框架。  在博客目录下打开Git Bash，在Hexo官网 里依次输入安装指令，即可完成安装。 12345npm install hexo-cli -ghexo init blogcd blognpm insta

https://aplas-plus.github.io/Techonology/%E5%9F%BA%E4%BA%8EHexo%E6%A1%86%E6%9E%B6%E6%90%AD%E5%BB%BA%E4%B8%AA%E4%BA%BA%E5%8D%9A%E5%AE%A2%E8%AE%B0%E5%BD%95/

**Comment by Li-Hongyu** [comment]
你的文字，清晰而深刻， 解开谜团，点亮黑暗。 你的博客，是我学习的灯塔， 在编程的海洋中，我不再迷茫。
